massa-protocol-exports/Cargo.toml

```diff
@@ -10,7 +10,7 @@ thiserror = "1.0"
 nom = "7.1"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-peernet = { git = "https://github.com/massalabs/PeerNet", rev = "0ad0e2e" }
+peernet = { git = "https://github.com/massalabs/PeerNet", branch = "debuig" }
```

Comment:
maybe provide commit revision instead of branch ?

Reply:
Done